modules/admin/admin-api/src/main/java/com/enonic/xp/admin/widget/WidgetDescriptor.java

```diff
@@ -37,7 +37,7 @@ public class WidgetDescriptor
 
     private final ImmutableMap<String, String> config;
 
-    private static final String URL_PREFIX = "_/widgets/";
+    private static final String URL_PREFIX = "_/widget/";
```

Comment:
Deprecate/remove getUrl method?

Reply:
It will be done in https://github.com/enonic/xp/issues/10672